text/0232-maturity-of-otel.md

```diff
@@ -0,0 +1,153 @@
+# Framework for defining the maturity of OpenTelemetry and its SIGs
+
+On 08 Mar 2023, the OpenTelemetry GC and TC held an OpenTelemetry Leadership summit, discussing various topics. One of the themes we discussed was establishing standard rules for describing the maturity of the OpenTelemetry project. This OTEP summarizes what was discussed there and is intended to have the wider community provide feedback.
+
+This OTEP builds on what was previously communicated by the project, especially the [Versioning and stability for OpenTelemetry clients](https://opentelemetry.io/docs/reference/specification/versioning-and-stability).
+
+The Collector's [stability levels](https://github.com/open-telemetry/opentelemetry-collector#stability-levels) inspired the maturity levels.
+
+## Motivation
+
+Quite often, the community is faced with the question of the quality and maturity expectations of its diverse set of components. This OTEP aims to bring clarity by establishing a framework to communicate the project's maturity. As the OpenTelemetry project comprises a multitude of SIGs, and each SIG has several components of varying quality, having this framework will help set the right expectations for OpenTelemetry users. The public message we want to convey by stating that a SIG is a tier-1 SIG is that the SIG has both a healthy community and is seen as critical to the project as a whole.
+
+Only once we reach the "stable" level at the project level will we be ready to consider graduating OpenTelemetry at the CNCF, with all of the Tier-1 SIGs being in the context of the graduation, including their Tier-1 Components. This OTEP provides a framework for describing project maturity as a whole.
+
+## Glossary
+
+* *SIG:* an official SIG within the OpenTelemetry project, like the Collector or Java. Given that some Working Groups (WGs) are effectively SIGs, like the Semantic Conventions WG, WGs will be accepted on an individual basis.
```

Comment:
I'm not convinced Semantic Conventions WG (as the charter implies) is a SiG.  It could be we need to evolve into one, but that would be a change of charter.  Instead, we're attempting to grow individual ownership of sub-WG.

I think I understand the goals of stability here, but I don't want to implicitly be changing the charter of a working group.  Specifically the current SemConv WG is meant to finish process + tooling, at which point it can re-evaluate its charter and existence.

Reply:
That's great feedback, I did think the Semantic Conventions WG was not properly classified and should be a SIG. Other folks felt the same, but it's good to see that it's not the case.

Do you think I should leave the semantic conventions out of the otep for now?